src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -153,18 +158,28 @@ export const ContentMessageComponent = ({
 
   const isConversationReadonly = conversation.readOnlyState() !== null;
 
+  const contentMessageWrapperRef = (element: HTMLDivElement | null) => {
+    setTimeout(() => {
+      if (element?.parentElement?.querySelector(':hover') === element) {
+        // Trigger the action menu in case the component is rendered with the mouse already hovering over it
+        setActionMenuVisibility(true);
+      }
+    });
+  };
+
+  const asset = assets?.[0];
+  const isFileMessage = !!(asset as FileAssetType)?.isFile();
+  const isAudioMessage = !!(asset as FileAssetType)?.isAudio();
+  const isVideoMessage = !!(asset as FileAssetType)?.isVideo();
+  const isImageMessage = !!(asset as FileAssetType)?.isImage();
```

Comment:
**suggestion**: If there's really no way around casting, can we cast it once in line 170 instead of doing it 4 times?